### PR TITLE
feat: disable curation if items are being uploaded

### LINKED ISSUE #1459: Prevent committee members to approve a collection if the item is not in the catalysts

### PR DESCRIPTION
**Pull request by nicosantangelo**
Closes #1459

### CODE REVIEW

File: src/modules/translation/languages/en.json [hunk 1]
```diff
@@ -1020,6 +1020,7 @@
       "discussion": "Discussion",
       "keep_reviewing": "Keep reviewing",
       "back_to_curation": "Back to curation",
+      "waiting_for_upload": "Some items in the collection are still being uploaded. Please check back soon to proceed with the curation.",
```
Comment (nachomazzara) [inline src/modules/translation/languages/en.json:1023]: What do you think of mention where the item needs to be uploaded?

`[...] still being uploaded to the catalysts`

Comment (nicosantangelo) [reply src/modules/translation/languages/en.json:1023]: I wasn't sure the user knew what that meant (there's no mention of catalysts on the entire site). Maybe is a good idea anyways? I'm not sure if members of the committee know what a Catalyst is (besides...shibu)

Comment (nachomazzara) [reply src/modules/translation/languages/en.json:1023]: I guess that will help them to know about the catalyst and also will be easy to report

Comment (nicosantangelo) [reply src/modules/translation/languages/en.json]: agree, ok! added

**Pull request merged**